Description: avoid apparmor host name resolution issues
 In certain cases libvirt checks for host name resolution but is denied by
 apparmor as of today. This error is non-fatal, but was reported to cause quite
 some long deleays.
Author: Christian Ehrhardt <christian.ehrhardt@canonical.com>
Forwarded: not yet (major rework to sync apparmor changes planned)

--- a/examples/apparmor/usr.lib.libvirt.virt-aa-helper.in
+++ b/examples/apparmor/usr.lib.libvirt.virt-aa-helper.in
@@ -4,6 +4,9 @@
 /usr/lib/libvirt/virt-aa-helper {
   #include <abstractions/base>
   #include <abstractions/user-tmp>
+  # Avoid name resolution issues (LP: #1546674)
+  #include <abstractions/nameservice>
+
   # Site-specific additions and overrides. See local/README for details.
   #include <local/usr.lib.libvirt.virt-aa-helper>
 
